hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/scheduler/HdfsNode.java

```diff
@@ -28,19 +26,20 @@
  */
 public abstract class HdfsNode implements IOfferEvaluator, ILauncher {
   private final Log log = LogFactory.getLog(HdfsNode.class);
-  private final LiveState liveState;
   private final ResourceFactory resourceFactory;
 
   protected final HdfsFrameworkConfig config;
-  protected final IPersistentStateStore persistenceStore;
+  protected final HdfsState state;
```

Comment:
I don't like that HdfsState is not an interface... I would rather make this an injectable impl fronted by an interface.